frappe/integrations/doctype/google_drive/google_drive.json

```diff
@@ -0,0 +1,119 @@
+{
+ "creation": "2019-08-13 17:24:05.470876",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "enable",
+  "sb_00",
+  "user",
+  "backup_folder_name",
+  "frequency",
+  "authorize_google_drive_access",
+  "column_break_5",
+  "backup_folder_id",
+  "last_backup_on",
+  "refresh_token",
+  "authorization_code"
+ ],
+ "fields": [
+  {
+   "default": "0",
+   "fieldname": "enable",
```

Comment:
Use `enabled` or `disabled`, they are standard fieldnames.

```suggestion
   "fieldname": "enabled",
```

Reply:
Its `enable` in all other Google Integrations, so it wouldnt make sense to keep `enabled` in this one.